### ISSUE TITLE
jsdoc of enum values

### ISSUE THREAD

**Issue by benny-medflyt** [dphilipson/typescript-string-enums#7]
This is a cool little library.

When manually hand-rolling an enum, it is possible to annotate each enum value with a jsdoc. This way, your IDE can show you documentation during autocomplete and during hover:

<img width="510" alt="tmp00001" src="https://cloud.githubusercontent.com/assets/24827547/23700611/fc673f56-03fb-11e7-9f2d-17a78ed60487.png">

Is it possible to achieve this also with `typescript-string-enums`?

**Comment by dphilipson** [comment]
Hi benny, first of all, sorry for the slow response.

Unfortunately I do not know of a way to make this work. I would be surprised if this is possible, as the jsdoc would need to understand to make the jump from a function argument to the key of a computed type. If documenting individual values is important to you, you may be stuck with the repetitive definition.

**Comment by dphilipson** [comment]
Fixed in `0.2.0` with the new object syntax. The following now works:

``` javascript
export const Status = Enum({
    /**
     * Everything is fine.
     *
     * Hovering over Status.RUNNING in an IDE will show this comment.
     */
    RUNNING: "running",

    /**
     * All is lost.
     */
    STOPPED: "stopped",
});
export type Status = Enum<typeof Status>;

console.log(Status.RUNNING); // -> "running"
```
